wlog-zap/internal/marshalers/wtracing_spanmodel.go

```diff
@@ -49,10 +49,30 @@ func marshalWTracingSpanModel(key string, val interface{}) zapcore.Field {
 				}
 			}
 		}
+		if tags := span.Tags; tags != nil && len(tags) > 0 {
```

Comment:
check `len(tags)>0` might seem weird here but the zap span allocates the map object whether or not we gave it tags so when it comes back to the logger `span.Tags` is always non-null. We want the behavior of not printing empty tags so we need to check the length here.

Reply:
Thanks for the context. You can actually reduce this down to just the `len(tags) > 0` since `len(nil) == 0`.